src/test/java/io/crate/jmx/integrationtests/MetricsITest.java

```diff
@@ -68,9 +68,22 @@ public void testQueryStatsMetrics() {
     public void testConnectionsMetrics() {
         assertMetricValue("crate_connections{protocol=\"psql\",property=\"open\",} ");
         assertMetricValue("crate_connections{protocol=\"psql\",property=\"total\",} ");
+        assertMetricValue("crate_connections{protocol=\"psql\",property=\"messagesreceived\",} ", -1);
```

Comment:
In order to create psql connections we need psql JDBC driver, and my argument against this is that we may need to handle different versions to test against different CrateDB versions. If you think it's important to get actual values > -1 here, I can give it a try.

Reply:
We do not really need to test what values are returned here. We just want to test that the metric exists and contains a numeric value. I suggest to just adjust the `assertMetricValue` to compare against `-1` by default, I see no need to make this configurable.

Reply:
:+1